source/templates/conditionals.md

```diff
@@ -1,7 +1,49 @@
-Sometimes you may only want to display part of your template if a property
-exists.
+Conditional statements bring a minimum of logic into Ember templating.
+Statements like `if` and `unless` are implemented as built-in helpers. Helpers
+can be invoked three ways.
```

Comment:
The first paragraph reads:

> Conditional statements bring a minimum of logic into Ember templating.
> Statements like `if` and `unless` are implemented as built-in helpers. Helpers
> can be invoked three ways.

Conditional helpers provide a nice contextual way to introduce the different usages, because they are so universally useful. I'm not sure what to change here. Adding a section "using Ember helpers" could precede conditionals, but would probably still want to cover conditionals as the example use-case. That or talk about adding custom helpers but, that move seems aggressive to me.

Perhaps this section could be re-named "Using Helpers"? It defaults in the other direction away from conditionals, but it perhaps more of a useful title.